sql/hive/src/test/scala/org/apache/spark/sql/hive/orc/HiveOrcQuerySuite.scala

```diff
@@ -316,4 +317,33 @@ class HiveOrcQuerySuite extends OrcQueryTest with TestHiveSingleton {
       }
     }
   }
+
+  test("SPARK-32864: Support ORC forced positional evolution") {
+    Seq("native", "hive").foreach { orcImpl =>
+      Seq(true, false).foreach { forcePositionalEvolution =>
+        withSQLConf(SQLConf.ORC_IMPLEMENTATION.key -> orcImpl,
+          OrcConf.FORCE_POSITIONAL_EVOLUTION.getAttribute -> forcePositionalEvolution.toString) {
```

Comment:
We want to use this config as it is? Or append usual prefix `spark.hadoop`?

Reply:
I recall we add a prefix for third-party library configs like mentioned above. Is this an exception?

Reply:
Well, I think you might be right, I've prefixed the config in https://github.com/apache/spark/pull/29737/commits/fbab1e4c597ad2fede82892935e665832217ce1b. Let me know it is ok.